server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java

```diff
@@ -1505,25 +1506,7 @@ private void parseIndexPredicate(IndexPredicateClauseContext ctx) {
     } else {
       path = parseFullPath(ctx.fullPath());
     }
-    if (ctx.LIKE() != null) {
```

Comment:
seems conflict with the index query, maybe you could chose another key word like match

Reply:
Now,if it is a likeIndex, ANTLR will be resolved to indexpredicateclause. If it is a fuzzy query, ANTLR will be resolved to predict